authorization/src/lib.rs

```diff
@@ -46,12 +44,11 @@ impl NonceAuth for WrappedAuth {
     }
 }
 
-pub struct AuthContract;
+pub struct ExampleContract;
```

Comment:
The first time I read the example I thought this was a contract that provided authorization as a service. I realize now it is a contract that is an example of authorization. I think other people might make this mistake, and a rename here should help.